src/until.js

```diff
@@ -0,0 +1,25 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ *
+ * @func
+ * @memberOf R
+ * @category Logic
+ * @sig (a -> Boolean) -> (a -> a) -> a -> a
+ * @param {Function} pred A predicate function
+ * @param {Function} fn The iterator function
+ * @param {*} init Initial value
+ * @return {*} Final value that satisfies predicate
+ * @example
+ *
+ *      R.until(R.gt(R.__, 100), R.multiply(2))(1) // => 128
+ *      R.until(R.none(R.isArrayLike), R.unnest)([1, [2], [[3]]]) // => [1, 2, 3]
```

Comment:
Let's remove this example as [`R.flatten`][1] would be much clearer. ;)

The first example is great!


[1]: http://ramdajs.com/docs/#flatten